docs/guides/dao-members/proposals.md

```diff
@@ -160,31 +160,42 @@ To create a new proposal using the DAO dashboard:
 
    > Defines the signature of the function to call within the target contract.
    > For the target contract USDC mentioned above use
+
+   <!-- WARNING: A space wil cause the transaction to fail in solidity. -->
+
    > `transfer(address,uint256)`. <span style="color:red">Do not use any spaces
    > in the signature: leading, trailing or otherwise.</span>
 
 7. Enter an **ETH Value**.
 
    > You can use zero if the target function is not `payable`.
 
-8. Enter **Parameters** which are the arguments that will be used to satisfy the
-   signature of the target contract function.
+8. In the case of step #5 above, the address
+   `0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48` is for the USDC contract. You
+   will be calling its `transfer` function, as indicated in step #6, should the
+   proposal pass. Enter **Parameters** which are the arguments that will be used
+   to satisfy the signature of the target contract function in step #6.
 
    > The arguments must be provided in JSON array format where the values are
-   > stringified.
+   > stringified. `["0xF4EB52Cf9D31a...d1663d78ddDEE9","499999000000"]`
 
-<!-- prettier will add a space between the elements in the array.
-A space wil cause the transaction to fail in solidity. -->
-<!-- prettier-ignore -->
-   ```json
-   ["0xF4EB52Cf9D31a...d1663d78ddDEE9","499999000000"]
-   ```
+   In the example directly above, the respective Agent (primary or secondary )
+   would be calling the USDC contract
+   (`0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48`) to transfer 499,999 USDC to
+   `0xF4EB52Cf9D31a...d1663d78ddDEE9`. Note that since
+   `transfer(address,uint256)` function transfers funds from the sender to the
+   specified address, the USDC is asked to be supplied from the Agent's balance.
 
-In the example above, the respective Agent (primary or secondary) would be
-calling the USDC contract (`0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48`) to
-transfer 499,999 USDC to `0xF4EB52Cf9D31a...d1663d78ddDEE9`. Note that since
-`transfer(address,uint256)` transfers funds from the sender to the specified
-address, the USDC is asked to be supplied from the Agent's balance.
+   USDC uses 6 decimal places of precision as opposed to 18 that many other
+   ERC20 tokens use. Consequently, multiply the desired $USD by 10^6 to get the
+   USDC amount to enter.
+
+   ```
```

Comment:
I understand the intent, but I'd leave out this table while keeping the text above it. Different geographies treat comma and period differently in numbers and there is already the example of 499,999.

Reply:
I am a big fan of examples but do see this one as overkill.